src/containers/VisualElement/VisualElementSearch.jsx

```diff
@@ -161,7 +161,6 @@ class VisualElementSearch extends Component {
             onSelect={h5p =>
               handleVisualElementChange({
                 resource: 'h5p',
-                path: h5p.path,
```

Comment:
Denne kan vel ikkje slettes. I artikkel så er det path som må lagres. data-url er regna ut på bakgrunn av dette feltet.

Reply:
Dette er den nye pathen, som jeg sletter fordi du ikke ville ha den nye pathen, men den gamle

Reply:
Ah. Sjølvsagt.

Reply:
Problemet er at samme koden brukes for å sette inn ny h5p i artikkel også. Ved å fjerne dette så fungerer det ikkje lenger å sette inn h5p i artikkel. Du må skille mellom innsetting av ny h5p og redigering av gammel.

Reply:
hm. ok tror jeg bare må slette den gamle koden i DisplayExternal i så fall, og skrive den om.